src/axolotl/train.py

```diff
@@ -495,6 +496,9 @@ def train(
         processor,
     ) = setup_model_and_trainer(cfg, dataset_meta)
 
+    plugin_manager = PluginManager.get_instance()
+    plugin_manager.post_model_load(cfg, model)
```

Comment:
This is is called twice. I prefer the other call at the end of the `load_model` function.